docs/introduction.md

```diff
@@ -0,0 +1,144 @@
+![GeoStyler Logo](./Geo_Styler_Logo_300_RGB.jpg)
+
+GeoStyler is an Open Source React library that enables users to style maps with the help of a GUI.
+GeoStyler works with multiple styling formats and is highly customizable.
```

Comment:
Yo may want to add a link to the geostyler repo and npm package of the geostyler itself here.

Reply:
done